src/coverlet.core/CoverageDetails.cs

```diff
@@ -8,7 +8,18 @@ public class CoverageDetails
         public int Total { get; internal set; }
         public double Percent
         {
-            get => Math.Round(Total == 0 ? 1 : Covered / Total, 3);
+            get => Math.Round(Total == 0 ? 1 : Covered / Total, 4);
+        }
+
+        public double GetCoveragePercentage()
+        {
+            double percentage = Percent * 100;
+            return RoundDown(percentage);
+        }
+
+        private double RoundDown(double percentage)
```

Comment:
Can you avoid `RoundDown` helper and use inline directly on `CoveragePercentace` prop, if I'm not mistaken this helper is private and used only in one method so it's not so useful extract an helper method.

Reply:
Yes, its used only in one place as of now. 

I forget to ask this earlier, Average Coverage Percentage value could go more than 2 decimal precision since its calculated inline. 
`coverageTable.AddRow("Average", $"{totalLinePercent / numModules}%", $"{totalBranchPercent / numModules}%", $"{totalMethodPercent / numModules}%");`
Should I Floor the value with 2 decimal precision there also? Should I create `MathHelper` class under Helper folder, add this `RoundDown `method in the class & refer it in `CoveragePercentage `property and also during Average calculation? Is this fine?

Reply:
@MarcoRossignoli I have other changes ready and also before & after report formats output. 

I am waiting to see reply on what to do with "Average Percentage" calculation and then check-in code fully at once.

Reply:
I did deeper review...and I think that we should change a bit the idea.
We should avoid to change `CoverageDetail` object only to "round data", I don't like the idea to have members with same meaning but only different "formatting" seems a duplication to me, I mean `GetCoveragePercentage` is `Percent` formatted.
I like the idea of `double MathHelper.RoundPercentage(double)` in a file MathHelper.cs under Helpers folder and apply this helper in every place we need, in this way we can also apply it on `Average Coverage Percentage`.
For instance 
```
var totalLinePercent =  MathHelper.RoundPercentage(summary.CalculateLineCoverage(result.Modules).Percent * 100);
...
var method = MathHelper.RoundPercentage(summary.CalculateMethodCoverage(module.Value));
...
 coverageTable.AddRow("Average", $"{MathHelper.RoundPercentage(totalLinePercent / numModules)}%",...
```
In this way we have also cetralized "percentage rounding" that we can apply in a uniform way in all code base project.

Reply:
could be cute to have an extension method
```
namespace Coverlet.Core.Helpers
{
...
public static double RoundPercentage(this double value)
...
```

Reply:
Cool :) .. I too like the idea of extension method.

I will rewrite and submit the code tomorrow then.

Reply:
No rush take your time thank's and sorry for "change of direction"

Reply:
@sasivishnu @MarcoRossignoli it'll probably be more straightforward to just update the `Percent` property to automatically multiply by 100 and round down. I don't think there's really any need to introduce a new `GetCoveragePercentage` method. Regarding calculation for the Average Coverage Percentage, you can leave it how it is now, I'll work on it once this PR is merged as I need to address #346

Reply:
@MarcoRossignoli no issues. I was also deliberating while fixing since it can be done in multiple ways. It needed this kind of discussion.

@tonerdo yes, that will be straight forward. 

Then, we need to make change in Cobertura Report on "line-rate" and "branch-rate" column.

Before: 
```
coverage.Add(new XAttribute("line-rate", summary.CalculateLineCoverage(result.Modules).Percent.ToString()));
coverage.Add(new XAttribute("branch-rate", summary.CalculateBranchCoverage(result.Modules).Percent.ToString()));
```

After:
```
coverage.Add(new XAttribute("line-rate", (summary.CalculateLineCoverage(result.Modules).Percent / 100).ToString()));
coverage.Add(new XAttribute("branch-rate", (summary.CalculateBranchCoverage(result.Modules).Percent / 100).ToString()));
```

Its not going to cause any impact. Since we already rounded down, dividing by 100 is not going to make any new issue. 

If all is fine, then I will make a change in `Percent `property itself.

Reply:
The idea of helper was for future possible use of rounding percentage everywhere in same way, btw I'm ok also with this with "fix" where we need like above.

Reply:
I am planning to do both. Use `Percent `property only but add extension method in new `MathHelper.cs` class as `public static double RoundPercentage(this double value)`.

Reply:
@sasivishnu avoid helper at all, less code to review and maintain, if in future we'll need to reuse we'll move to helper

Reply:
Thanks @sasivishnu, let me know when you'd done this so I can merge. The rest of the code seems good to me

Reply:
@tonerdo Yep, I am done with my changes and pushed latest one.